### ISSUE TITLE
How to use a known modem's /dev file?

### ISSUE THREAD

**Issue by whitedavidp** [thess/callattendant#3]
I know that my modem (at least for testing) is connected to a serial port /dev/ttyS2. I can see that it searches for and finds/uses this. Is there some way to just have that used instead of the searching? Thanks

**Comment by thess** [comment]
Good suggestion...

**Comment by whitedavidp** [comment]
Thanks. I can simply do it myself (at least for my own installation) but first I have to figure out what to do and where :-)

Looks like it might be a simple modification to this line in hardware/modem.py:

com_ports_list = "/dev/ttyS2"

But I have not been able to test it yet. Cheers!

**Comment by whitedavidp** [comment]
Well, I have done some testing and I get the expected result when using this line:

com_port_list = b'/dev/ttyS2',

Obviously, this would make sense to place in the app config somewhere. But this is fine for me now. I have way far to go still.

Thanks

**Comment by thess** [comment]
Added new config:
```
# Optional modem serial port (comma separated list permitted)
#   If not specified, the modem will be auto-detected
#   Example: "/dev/ttyUSB0, /dev/ttyACM0"
MODEM_DEVICE = ""
```
closed by: daaa06617642b9fdfc0fae724da80b5915a8a7d6
